manifests/manifest_dev.yml

```diff
@@ -3,7 +3,7 @@ inherit: manifest_base.yml
 host: fec-dev-api
 services:
   - fec-creds-dev
-  - fec-redis-dev
+  - fec-redis
```

Comment:
Sorry to nitpick, they can all go in `manifest_base` now, as they are all the same.

Reply:
@vrajmohan updated :)